release-notes/README.md

```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## May 18, 2022
+
+### Android Campaign Classic 1.0.2
+
+* Android CampaignClassic SDK is now Adobe Campaign Classic (ACC) v8 compatible! BroadlogId can be provided in the UUID format in the notification tracking APIs.
```

Comment:
Should have a space between Campaign and Classic, right?

Reply:
After adding the space, the second usage of Adobe Campaign Classic could be replaced with just the `ACC` abbreviation.

Reply:
I think BroadlogId should be Broadlog ID

Reply:
{quote}
After adding the space, the second usage of Adobe Campaign Classic could be replaced with just the ACC abbreviation.
{quote}
The first usage is not Adobe Campaign Classic so I'll leave the expanded version, also it keeps it consistent with iOS release notes.

Reply:
Thanks for the quick review! I've updated the pr.